sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_models.py

```diff
@@ -61,6 +61,23 @@ def _to_generated_model(self):
         return jwk
 
 
+class KeyExportParameters(object):
+    """The parameters for a key export.
+
+    This includes a key and algorithm that are used for encrypting the exported key material.
+
+    :param key: The export encryption key. This must be an RSA key that supports encryption.
+    :type key: ~azure.keyvault.keys.JsonWebKey
+    :keyword str algorithm: The encryption algorithm to use to protected the exported key material. Possible values
+     include: "CKM_RSA_AES_KEY_WRAP", "RSA_AES_KEY_WRAP_256", "RSA_AES_KEY_WRAP_384".
+    """
+
+    def __init__(self, key, **kwargs):
```

Comment:
As mentioned above, we probably want to add `name` and the optional `version` to this.